cloudformation/PolicyTemplates/01-cis-developer-policy.yml

```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Creates developer policy for CIS Resources.  Grants admin over kinesis, lambda, and DynamoDb when appropriately tagged."
+Resources:
+  CISDeveloper:
+    Type: "AWS::IAM::ManagedPolicy"
+    Properties:
+      Description: Allows developers working on change integration admin level access to resources with specific tags.
+      Path: /
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          -
+            Effect: Allow
+            Action: lambda:*
+            Resource: "arn:aws:lambda:*:*:function:cis_functions*"
```

Comment:
The lambda function names always start with the project name through declared convention.  We would have to override the behavior in apex's config to change this.  So it seemed fine to me.